src/pixel_fifo.md

```diff
@@ -9,7 +9,7 @@ shortened to make up for the additional time in mode 3, as shown in the followin
 
 :::
 
-![](imgs/game-boy-lcd-refresh-diagram-2.svg "imgs/game-boy-lcd-refresh-diagram-2.svg")
+{{#include imgs/ppu_modes_timing.svg:2:}}
```

Comment:
What is this syntax? To get the inline?

Reply:
[Yep.](https://rust-lang.github.io/mdBook/format/mdbook.html#including-files)